docs/core/introduction.md

```diff
@@ -1,57 +1,379 @@
+
+## Microsoft.Extensions libraries
+
+Libraries for some commonly used application functionality aren't included in the runtime libraries but are made available in NuGet packages, such as the following:
+
+| NuGet package | Documentation |
+|---------|---------|
+| [Microsoft.Extensions.Hosting](https://www.nuget.org/packages/Microsoft.Extensions.Hosting) | [Application lifetime management (Generic Host)](extensions/generic-host.md) |
+| [Microsoft.Extensions.DependencyInjection](https://www.nuget.org/packages/Microsoft.Extensions.DependencyInjection) | [Dependency injection (DI)](extensions/dependency-injection.md)
+| [Microsoft.Extensions.Configuration](https://www.nuget.org/packages/Microsoft.Extensions.Configuration) | [Configuration](extensions/configuration.md) |
+| [Microsoft.Extensions.Logging](https://www.nuget.org/packages/Microsoft.Extensions.Logging) | [Logging](/aspnet/core/fundamentals/logging) |
+
+For more information, see the [dotnet/extensions repository on GitHub](https://github.com/dotnet/extensions).
+
+## Data access
+
+.NET provides an Object/Relational Mapper (ORM) and a way to write SQL queries in code.
+
+### Entity Framework Core
+
+Entity Framework (EF) Core is an [open source](https://github.com/aspnet/EntityFrameworkCore) and cross platform data access technology that can serve as an ORM. EF Core lets you work with a database by referring to .NET objects in code. It reduces the amount of data-access code you would otherwise need to write and test. EF Core supports many database engines.
+
+For more information, see [Entity Framework Core](/ef/core/) and [Database Providers](/ef/core/providers/).
+
+### LINQ
+
+Language-integrated query (LINQ) lets you write declarative code for operating on data. The data can be in many forms (such as in-memory objects, a SQL database, or an XML document), but the LINQ code you write typically doesn't differ by data source.
+
+For more information, see [LINQ (Language Integrated Query) overview](../standard/linq/index.md).
+
+## .NET terminology
+
+To understand .NET documentation, it can help to know how the usage of some terms has changed over time.
+
+### .NET Core and .NET 5
+
+In 2002, Microsoft released [.NET Framework](../framework/get-started/overview.md), a development platform for creating Windows apps. Today .NET Framework is at version 4.8 and is still [supported by Microsoft](https://dotnet.microsoft.com/platform/support/policy/dotnet-framework).
+
+In 2014, Microsoft began writing a cross platform open source successor to .NET Framework. This new implementation of .NET was named .NET Core until it reached version 3.1. The next version after .NET Core 3.1 is .NET 5, currently in preview. Version number 4 was skipped to avoid confusion between this implementation of .NET and .NET Framework 4.8. The name "Core" was dropped to make clear that this is now the main implementation of .NET.
```

Comment:
```suggestion
In 2014, Microsoft began writing a cross-platform, open-source successor to .NET Framework. This new implementation of .NET was named .NET Core until it reached version 3.1. The next version after .NET Core 3.1 is .NET 5.0, which will be released in November 2020. Version number 4 was skipped to avoid confusion between this implementation of .NET and .NET Framework 4.8. The name "Core" was dropped to make clear that this is now the main implementation of .NET.
```